aexpect/client.py

```diff
@@ -1240,7 +1240,8 @@ def cmd_status_output(self, cmd, timeout=60, internal_timeout=None,
             raise ShellStatusError(cmd, out)
 
         # Get the first line consisting of digits only
-        digit_lines = [l for l in status.splitlines() if l.strip().isdigit()]
+        digit_lines = [l for l in status.splitlines()
+                       if re.match("^-?[0-9]+$", l.strip())]
```

Comment:
good catch. Would you please compile the regexp and store it as `self.RE_STATUS` so it doesn't have to be compiled per each line.

Reply:
... or rather `self.__RE_STATUS`

Reply:
Sure, thanks.